src/app/blog/index.html

```diff
@@ -3,7 +3,34 @@
   pageTitle="SKY UX Blog"
   windowTitle="SKY UX Blog">
   
-<article>
+	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — March 2018</h2>
+			<p>SKY UX releases 2.10.0-2.11.0</p>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized"><a routerLink="/components/lookup">Lookup</a></h4>
+					<p>The lookup component combines typeahead search functionality with the option to select one or multiple items for data entry.</p>
+				<h4 class="sky-emphasized"><a routerLink="/components/flyout">Flyout</a></h4>
+					<p>Added the ability to resize the flyout panel.</p>
+				<h4 class="sky-emphasized"><a routerLink="/components/alert">Alert</a>, <a routerLink="/components/label">Label</a>, and <a routerLink="/components/status-indicator">Status indicator</a></h4>
+					<p>Updated the styling of these three components to improve consistency and accessibility.</p>
+				<h4 class="sky-emphasized">UX guidelines</h4>
+			<p>Updated the UX guidelines for <a routerLink="/design/guidelines/error-handling">error handling</a> and <a routerLink="/design/guidelines/form-design">form design</a></p>
```

Comment:
Should this also list the updated UX guidelines for inline help in https://github.com/blackbaud/skyux2-docs/pull/56?